meerdag/test/meerdag_test.go

```diff
@@ -149,7 +149,24 @@ var testDataFilePath string = "./testData.json"
 
 var tbMap map[string]meerdag.IBlock
 
+func getConfig(memory bool) *config.Config {
```

Comment:
I think it would be best to refactor configuration settings to support in-memory database. Although the `bool` flag is currently ok.

Reply:
> I think it would be best to refactor configuration settings to support in-memory database. Although the `bool` flag is currently ok.

In-memory mode only appears in testing